tests/e2e/pageobjects/ide/CheCodeLocatorLoader.ts

```diff
@@ -36,6 +36,11 @@ export class CheCodeLocatorLoader extends LocatorLoader {
                     button: By.xpath('//a[contains(., "trust")]')
                 },
             },
+            extras: {
```

Comment:
`ExtensionsViewSection` looks like regular locator

If there is nothing special, I would suggest to move it to locators, as more relevant place.

Reply:
https://github.com/eclipse/che/blob/3a7b3857ced5d8c407379046756196b1e0bc80ff/tests/e2e/pageobjects/ide/CheCodeLocatorLoader.ts#L19

Reply:
Got it, thanks.